packages/@sanity/code-input/src/Schema.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  name: 'code',
+  type: 'object',
+  fields: [
+    {
+      title: 'Code',
+      name: 'code',
+      type: 'text'
+    },
+    {
+      name: 'language',
+      title: 'Language',
+      type: 'string'
+    },
+    {
+      title: 'Highlighted lines',
+      name: 'highlightedLines',
+      type: 'array',
+      of: [{
+        type: 'string',
+        title: 'Highlighted line'
```

Comment:
I got errors when trying to load the data without the code-input component, but this will maybe work when me merge inn the new branch where we can have array of numbers. Array of numbers is best 👍